vertical-pod-autoscaler/pkg/recommender/input/history/history_provider.go

```diff
@@ -26,10 +26,22 @@ import (
 	"k8s.io/autoscaler/vertical-pod-autoscaler/pkg/recommender/model"
 )
 
```

Comment:
I have a doubt. Why struct  "PrometheusHistoryProviderConfig"  and "prometheusHistoryProviderConfig" have almost similar name and similar fields.